etc/make-hilla-badge-url.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+hilla_version=${1}
+
+if [[ -z "${hilla_version}" ]]; then
+    echo "Usage: $0 <hilla major.minor version>"
+    echo
+    echo "Example: $0 2.5"
+    echo
+    exit 1
+fi
+
+echo "URL to use with a URL shortener service like https://tinyurl.com/ to create a Hilla badge"
+echo "until Hilla gets added to https://simpleicons.org/"
```

Comment:
Maybe we should request it somehow?

Reply:
There's a link to asking for new icons https://github.com/simple-icons/simple-icons/issues/new?assignees=&labels=new+icon&template=icon_request.yml

But it should probably be done by the owners of the logo

Reply:
Can you bring this onto line ?

Reply:
Yes, it can be done, but I don't know if readability in console would be good